bin/process_backlog_study.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
I'm wondering if the name of the script is correct since it's only going to prepare the backlog study. 
Unless you plan to add the ingestion steps afterward.

Reply:
That's true, I'll rename it to `prepare_backlog_study`.